src/docfx/build/xref/XrefMap.cs

```diff
@@ -248,6 +252,13 @@ private static string GetSubstringFromContent(string content, int startLine, int
             var result = new StringBuilder();
             var currentLine = 1;
             var currentColumn = 1;
+
+            // for better performance by accessing index when content is 1 line
+            if (currentLine == startLine && currentLine == endLine)
+            {
+                return content.Substring(startColumn - 1, endColumn - startColumn + 1);
```

Comment:
Move this to L275? Then this optimization is not limited to currentLine == startLine == endLine **== 1**

Reply:
This only works for **1** line content. If `currentLine == startLine == endLine > 1` we do not know the index directly